script.js

```diff
@@ -0,0 +1,44 @@
+window.addEventListener('load', function() {
+    /* Write your solution between this comment */
+/*Heather Sleyster
+sleyster_assignment01b
+INFO 2134
+Thoendel
+03/08/2020
+*/
+
+/*These statements assign the select tag to the variable s 
+and the option tags to the o array.
+*/
+let s = document.createElement('select');

```

Comment:
the select already exists...I think you want to create an option element for every iteration of your loop below...